apps/bletiny/src/cmd.c

```diff
@@ -2464,6 +2464,46 @@ static struct kv_pair cmd_set_addr_types[] = {
 };
 
 static void
+bletiny_set_priv_mode_help(void)
+{
+    console_printf("Available set priv_mode params: \n");
+    help_cmd_kv_dflt("addr_type", cmd_set_addr_types, BLE_ADDR_PUBLIC);
+    help_cmd_byte_stream_exact_length("addr", 6);
+    help_cmd_uint8("mode");
+}
+
+static int
+cmd_set_priv_mode(void)
+{
+    ble_addr_t addr;
+    uint8_t priv_mode;
+    int rc;
+
+    addr.type = parse_arg_kv_default("addr_type", cmd_set_addr_types,
+                                     BLE_ADDR_PUBLIC, &rc);
+    if (rc != 0) {
+        console_printf("invalid 'addr_type' parameter\n");
+        help_cmd_kv_dflt("addr_type", cmd_set_addr_types, BLE_ADDR_PUBLIC);
+        return rc;
+    }
+
+    rc = parse_arg_mac("addr", addr.val);
+    if (rc != 0) {
+        console_printf("invalid 'addr' parameter\n");
+        help_cmd_byte_stream_exact_length("addr", 6);
+        return rc;
+    }
+
+    priv_mode = parse_arg_uint8("mode", &rc);
+    if (rc != 0) {
+        console_printf("missing mode\n");
+        return -1;
```

Comment:
Certainly not a big deal, but should this be `return rc` rather than `return -1`?